test/unit/region.build-region.spec.js

```diff
@@ -70,6 +77,8 @@ describe('Region', function() {
         beforeEach(function() {
           this.definition = {selector: this.fooSelector};
           this.region = Marionette.Region.buildRegion(this.definition, this.DefaultRegionClass);
+
+          delete this.region.cid;
```

Comment:
ugh, is this because of the deep equalities? 

I had to deal with this too at one point, I wonder if we could change the checks in regions.spec, to be a little friendlier...
like 
```js
sameRegion = function(ra, rb) {var props = _.partial(_.pluck, _, [list_of_props]), return props(ra) == props(rb) }
```